### ISSUE TITLE
Slow conversion after reset in Windows 11

### ISSUE THREAD

**Issue by UPSA007** [ciromattia/kcc#734]
I really don't know whether is it a bug in KCC or Windows but let me describe it. I have been using KCC frequently for  long time without any problems in windows 11. Recently I reset my windows. After the reset KCC started taking a long time to convert CBZ files. For example a CBZ with 32 pages 110 MB  takes around 10 minutes to convert to KFX epub with mozJpeg on where it used to take less than 2 minutes to convert. before. I tried both 6.1.1 and 6.1.0 version of KCC. I checked in task manager and noticed when some thing is converting KCC creates multiple processes (like 15 or 20) which do nothing at all (individual process CPU and Disk usage 0 RAM usage around 20MB). Then suddenly after 6 or 7 minutes all of them start working (individual process CPU 5%) and stop when conversion is finished. Can someone please help. Thank you in advance.

**Comment by axu2** [comment]
How did you reset windows 11? Did you download the win11 installer from the Microsoft website to do a clean reinstall?

**Comment by UPSA007** [comment]
No I did not do a clean install. Rest it using "reset this pc" option in windows. And can you please share a link of this new kfx workflow. Thank You.

**Comment by axu2** [comment]
A clean reinstall from USB will probably fix your issue. 

New kfx workflow:

- https://github.com/ciromattia/kcc/issues/460#issuecomment-2254682792

**Comment by UPSA007** [comment]
I saw the workflow but it seems too complicated. The quality is better no doubt. But can't beat the convenience of KCC epub kfx. Besides I generally upload the epub in "Send to Kindle".

**Comment by axu2** [comment]
Also some KCC settings can cause slow conversion, can you send a screenshot of settings? @UPSA007

**Comment by UPSA007** [comment]
Yeah you are right the reset was the cause. Did a clean install now everything is working fine. However I think I discovered the the root cause. After reset Win 11 was applying efficiency mode on the KCC processes.
